lib/codegen/generator.dart

```diff
@@ -290,9 +291,9 @@ class _DeclarationCode extends ConstExpression {
   _DeclarationCode(this.name, this.type, this.kind, this.isFinal, this.isStatic,
       this.annotations);
 
-  List<String> get librariesUsed => []
+  List<String> get librariesUsed => <String>[]
     ..addAll(type.librariesUsed)
-    ..addAll(annotations.expand((a) => a.librariesUsed));
+    ..addAll(new List<String>.from(annotations.expand((a) => a.librariesUsed)));
```

Comment:
now that we know this was an issue with analysis server, can change this back to use generics?